### PR TITLE
Calculator basic design

### PR DESCRIPTION
**Pull request by gabrielyea**
# Calculator app made with react.
- Added Calculator component
- Added Buttons components
- Added Display component.
- Implemented a basic click handler to show values on display.
- [Live Demo](https://gabrielyea.github.io/math-magicians/)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,10 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <header className="App-header" />
+      <Calculator />
```
Comment (arslanbisharat) [inline src/App.js:12]: - [x] App js should have class based component. Please work on it.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import './calculatorStyle.scss';
+import CalculatorButton from './buttons/CalculatorBtn';
+import Display from './display/Display';
+
+// eslint-disable-next-line react/prefer-stateless-function
+export default class Calculator extends React.Component {
```
Comment (arslanbisharat) [inline src/components/Calculator.js:6]: - [x] Calculator js should have class based component using ES6 syntax. Please work on it.
```

'import React, { Component } from 'react';

class Calculator extends Component {
  render() {
    // reference as a js object
    return {
      <div>
       // Calculator app design goes here
      </div>
    };
  }
}

export default Calculator;

```

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/Calculator.js (hunk 2 above)
Comment (gabrielyea) [reply src/components/Calculator.js:6]: Calculator already was a class

File: src/App.js (hunk 1 above)
Comment (henatan99) [reply src/App.js:12]: - This issue is resolved. :+1:

File: src/components/Calculator.js (hunk 2 above)
Comment (henatan99) [reply src/components/Calculator.js:6]: - This issue is resolved. Thanks for making the changes requested by the previous reviewer. :clap:

File: src/components/calculatorStyle.scss [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  gap: 0;
+  row-gap: 0;
+  margin: auto;
+  margin-top: 100px;
+  padding: 0;
+  width: 300px;
+  background-color: white;
+  grid-template-areas:
```
Comment (henatan99) [inline src/components/calculatorStyle.scss:16]: - [ ] OPTIONAL: I believe there is still something to do with your UI design compared to the given design. You can make changes when you have spare time. :dancers: 
image         
image

**Review by henatan99 — APPROVED** [review]
Hi @gabrielyea , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**